platform/android/export/export.cpp

```diff
@@ -1874,11 +1876,28 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			String package_name = get_package_name(p_preset->get("package/unique_name"));
 
 			Vector<PluginConfig> enabled_plugins = get_enabled_plugins(p_preset);
+			String plugin_names = get_plugins_names(enabled_plugins);
 			String local_plugins_binaries = get_plugins_binaries(BINARY_TYPE_LOCAL, enabled_plugins);
 			String remote_plugins_binaries = get_plugins_binaries(BINARY_TYPE_REMOTE, enabled_plugins);
 			String custom_maven_repos = get_plugins_custom_maven_repos(enabled_plugins);
 
+			// check if plugins have changed since last build
+			bool clean_build_required = plugin_names != last_plugin_names;
+			if (!clean_build_required) {
+				for (int i = 0; i < enabled_plugins.size(); i++) {
+					if (enabled_plugins.get(i).last_updated > last_custom_build_time) {
+						clean_build_required = true;
+						break;
+					}
+				}
+			}
+			last_custom_build_time = OS::get_singleton()->get_unix_time();
+			last_plugin_names = plugin_names;
+
```

Comment:
Nice implementation! Can you wrap the logic into an `inline` method. Since the surrounding method is frequently updated, this will ensure this logic is not inadvertently modified.

Reply:
Thanks! Yes, will do.